charts/pulsar/templates/oxia-coordinator-podmonitor.yaml

```
@@ -19,5 +19,5 @@
 
 # deploy oxia-coordinator PodMonitor only when `$.Values.oxia.coordinator.podMonitor.enabled` is true
 {{- if and $.Values.components.oxia $.Values.oxia.coordinator.podMonitor.enabled }}
-{{- include "pulsar.podMonitor" (list . "oxia.coordinator" "app.kubernetes.io/component: oxia-coordinator" "metrics") }}
-{{- end }}
\ No newline at end of file
+{{- include "pulsar.podMonitor" (list . "oxia.server" (printf "component: %s-coordinator" .Values.oxia.component)) }}
```

I guess `oxia.server` should be replaced with `oxia.coordinator` here:
```suggestion
{{- include "pulsar.podMonitor" (list . "oxia.coordinator" (printf "component: %s-coordinator" .Values.oxia.component)) }}
```

---

Thanks, this was a typo - it's fixed now.